src/Components/Profile/MyPosts/myPosts.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import Post from './Post/post';
+
+import style from './myPosts.module.css';
+
+const MyPosts = () => (
+  <div>
+    My posts
+    <div />
+    <textarea />
+    <button type="button">Add post</button>
+    <div className={style.posts}>
+      <Post message="Hi,How are you?" />
```

Comment:
I see there's a folder "Constants" on the src level.
Consider rahter creating a "core" folder with "constants.js" file inside and storing such strings there.
E.g. inside an exportable object named "I18Y" that stands for "Internationalisation" (I + 18 characters in between + N)

Reply:
I've done  all what you wrote.